src/test/java/data/CameraShotTest.java

```diff
@@ -11,22 +11,24 @@
 public class CameraShotTest {
     CameraShot cameraShot;
     @Before
-    public void initialize() {
+    public void initializeTest() {
         cameraShot = new CameraShot("camerashot-1");
     }
 
     @Test
-    public void getName() throws Exception {
+    public void getNameinitializeTest() {
```

Comment:
There is some missing camelcasing in the method naming here. Also, this just tests the `getName()` method, which is fine. The name of this test should be `getNameTest()` since the initialize method is already tested.